packages/dom/src/dom.js

```diff
@@ -529,28 +529,28 @@ export function isEntirelySelected( element ) {
 }
 
 /**
- * Given a DOM node, finds the closest scrollable container node.
+ * Given a DOM element, finds the closest scrollable container element.
  *
- * @param {Element} node Node from which to start.
+ * @param {Element} element Element from which to start.
  *
- * @return {?Element} Scrollable container node, if found.
+ * @return {Element|undefined} Scrollable container node, if found.
  */
-export function getScrollContainer( node ) {
-	if ( ! node ) {
+export function getScrollContainer( element ) {
```

Comment:
Changed to `element` since it expects a `Element` and not a `Node`

Reply:
> Changed to `element` since it expects an `Element` and not a `Node`

Technically `Element` implements the `Node` interface, so every element is a node ([reference](https://developer.mozilla.org/en-US/docs/Web/API/Element)). But for clarity's sake, I'd agree with your assessment 👍

Reply:
Correct. But [`clientHeight`](https://developer.mozilla.org/en-US/docs/Web/API/Element/clientHeight) and [`scrollHeight`](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollHeight) are only available on the `Element` interface so only an `Element` would be valid to pass into this function.